engine/install/binaries.md

```diff
@@ -173,6 +176,42 @@ The macOS binary includes the Docker client only. It does not include the
     This command downloads a test image and runs it in a container. When the
     container runs, it prints a message and exits.
 
+## Install server and client binaries on Windows
+
+> **Note**
+>
+> The following section describes how to install the Docker daemon on Windows Server which allows you to run Windows containers only. The binaries for Windows do not contain Docker components such as `buildx`, `docker scan`, and `docker compose`. If you are running Windows 10 or 11, we recommend that you install [Docker Desktop](../../desktop/index.md).
+
+Binary packages on Windows include both `dockerd.exe` and `docker.exe`. On Windows, these binaries only provide the ability to run native Windows containers (not Linux containers).
+
+To install server and client binaries, perform the following steps:
+
+1. Download the static binary archive. Go to
+    [https://download.docker.com/win/static/stable/x86_64](https://download.docker.com/win/static/stable/x86_64) and select the latest version from the list.
+
+2. Run the following PowerShell commands to install and extract the archive to your program files:
+
+    ```powershell
+    PS C:\> Expand-Archive /path/to/<FILE>.zip -DestinationPath $Env:ProgramFiles
+    ```
+
+3. Register the service and start the Docker Engine:
+
+    ```powershell
+    PS C:\> dockerd --register-service
```

Comment:
Small nit. We haven't added the new folder to the users PATH environment variable. 
Maybe we could do the simple workaround running dockerd with the absolute path:
```suggestion
    PS C:\> $Env:ProgramFiles\Docker\dockerd --register-service
```